src/python/module/affogato/segmentation/mws.py

```diff
@@ -4,8 +4,8 @@
 from ..affinities import compute_affinities
 
 
-def get_valid_edges(shape, offsets, number_of_attractive_channels,
-                    strides, randomize_strides, mask=None):
+def get_valid_edges(shape, offsets, number_of_attractive_channels=None,
+                    strides=None, randomize_strides=False, node_mask=None, edge_mask=None):
```

Comment:
What happens if we have both a `node_mask` and an `edge_mask`, does it result in a well defined state?
If not we should raise some exception int hat case.

Reply:
At the moment, when a `node_mask` is passed, all edges connected to a background node are masked. Then, if an `edge_mask` is also passed and there are extra edges that were not already masked, they are also masked.